product_code/lagom/user_service/api/src/main/scala/de/upb/cs/uc4/user/model/Address.scala

```diff
@@ -11,6 +11,15 @@ case class Address(street: String,
   def trim: Address = {
     copy(street.trim, houseNumber.trim, zipCode.trim, city.trim, country.trim)
   }
+
+  def oneEmpty: Boolean = {
+    if(street == "" || houseNumber == "" || zipCode == "" || city == "" || country == "") {
+      true
+    }
+    else {
```

Comment:
Reconsider the formatting of this line

Reply:
Can you give an idea, how you would format this, or what exactly should be reconsidered? Is it the position of the else?

Reply:
just write is as one line
```scala 
def oneEmpty: Boolean = {street == "" || houseNumber == "" || zipCode == "" || city == "" || country == ""}
```

Reply:
Yes, it is the position of the Else :smile: Is this a personal preference or recommented by the scala style guide?

Reply:
Fixed, closing conversation.